docs/sources/project/create-pr.md

```diff
@@ -56,28 +60,35 @@ Always rebase and squash your commits before making a pull request.
         pick 1a79f55 Tweak some of the other text for grammar
         pick 53e4983 Fix a link
         pick 3ce07bb Add a new line about RHEL
-        
-    If you run into trouble, `git --rebase abort` removes any changes and gets
-    you back to where you started. 
 
-4. Squash the `pick` keyword with `squash` on all but the first commit.
+    If you are not sure about the rebase at this point, delete all the lines and
```

Comment:
Rebasing is a tricky process and it is good to know that one can abort and restart the process. This is also the idea why `git rebase --abort` command is mentioned later on. In any case, I removed the sentence as well as mentioning of the abort command.